dev.yml

```diff
@@ -7,7 +7,7 @@ up:
     - mysql-client:
         or:        [mysql@5.7]
   - ruby:
-      version: 2.6.5
+      version: 3.1.2
```

Comment:
Should we leave this at 2.7? It'll make it easier to see that you're using a >2.7 feature without having to wait for CI.